### DISCUSSION TITLE
GitHub Copilot Chat v0.15.2 won't even let me open chat

### DISCUSSION THREAD

**Discussion by michaeltaman** [community/community#125435] [Copilot Conversations] [answered]
Product Feedback 

2024-05-23 21:59:33.469 [error] [auth] Failed to activate extension: "connect ETIMEDOUT 3.127.156.149:443". Chat failed to load. Please ensure that the GitHub Copilot Chat extension is up to date.
My Internet connection is working fine and all sites are accessible. Co-pilot only complains  Everything was working correctly until yesterday, and then suddenly stopped working the GitHub Copilot Chat v0.15.2 won't even let me open chat . Any restart does not help: restarting of computer, restarting VSCode, install/uninstall Copilot chat extension, Sigh out/ Sign In to GitHub account. Nothing not helps . Unable sign In via button appears.  I'm subscriber of Copilot. Payments regular without any delays.

**Comment by TrueMichato** [comment]
Experiencing the same issue as well, up until around 21 hours ago everything was working fine, now I am getting the exact same issue (`2024-05-24 13:05:06.193 [error] [auth] Extension activation failed: "connect ETIMEDOUT 3.127.156.149:443"`) and no amount of restarting and reinstalling seems to solve it.

**Comment by elimelechAdam** [comment]
Same

**Comment by hsenag** [comment]
Do you have any other extensions installed or a network setup that might be redirecting your traffic? `3.127.156.149` isn't part of the normal Copilot service.

**Reply by elimelechAdam** [reply]
i tried to uninstall all of the extenstions and install only copilot and im still getting the same error about the network not that i know

**Comment by canuckjacq** [comment, marked as answer]
~I'm noticing a pattern in the support tickets with this same error. For those in this thread, if you're happy to, can you share your country location and ISP?~

Nevermind - we've had some confirmation this appears to be an issue with a specific ISP. Please contact your ISP for assistance here.

**Comment by michaeltaman** [comment]
You can explain to your stationary network Internet provider that you are
experiencing a network error when trying to connect to a specific service
(GitHub Copilot) from your computer. Here's a suggestion on how to phrase
it:

"Hello, I've been experiencing issues with my internet connection when
trying to access a specific online service, GitHub Copilot, from my
computer. When I try to connect, I receive a network error message that
says 'Sign in failed: Error: network error. Extension activation failed:
"connect ETIMEDOUT 3.127.156.149:443"'. However, when I switch to using my
mobile network, I am able to connect to the service without any issues.
I've also tested accessing the service's API endpoint (
https://api.github.com) from both networks. It works fine on my mobile
network, but I'm unable to reach it on my stationary network. Could you
please help me troubleshoot this issue? I suspect it might be related to
some network settings or restrictions on my stationary network."

Remember to provide them with any additional details they might need, such
as the exact time the issue occurred, to help them diagnose the problem.
